packages/test/src/browser/view/test-view-contribution.ts

```diff
@@ -0,0 +1,300 @@
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { AbstractViewContribution, FrontendApplicationContribution, ViewContainerTitleOptions, Widget, codicon } from '@theia/core/lib/browser';
+import { Command, CommandRegistry, MenuModelRegistry, nls } from '@theia/core';
+import { inject, injectable } from '@theia/core/shared/inversify';
+import { TestItem, TestRunProfileKind, TestService } from '../test-service';
+import { ContextKeyService } from '@theia/core/lib/browser/context-key-service';
+import { TestTreeWidget } from './test-tree-widget';
+import { TabBarToolbarContribution, TabBarToolbarRegistry } from '@theia/core/lib/browser/shell/tab-bar-toolbar';
+import { TestCommandId } from '../constants';
+import { NavigationLocationService } from '@theia/editor/lib/browser/navigation/navigation-location-service';
+import { NavigationLocation } from '@theia/editor/lib/browser/navigation/navigation-location';
+import { FileService } from '@theia/filesystem/lib/browser/file-service';
+import { FileNavigatorCommands } from '@theia/navigator/lib/browser/file-navigator-commands';
+
+export namespace TestViewCommands {
+    /**
+     * Command which refreshes all test.
+     */
+    export const REFRESH: Command = Command.toDefaultLocalizedCommand({
+        id: TestCommandId.RefreshTestsAction,
+        label: 'Refresh Tests',
+        category: 'Test',
+        iconClass: codicon('refresh')
+    });
+
+    /**
+     * Command which cancels the refresh
+     */
+    export const CANCEL_REFRESH: Command = Command.toDefaultLocalizedCommand({
+        id: TestCommandId.CancelTestRefreshAction,
+        label: 'Cancel Test Refresh',
+        category: 'Test',
+        iconClass: codicon('stop')
+    });
+
+    export const RUN_ALL_TESTS: Command = Command.toDefaultLocalizedCommand({
+        id: TestCommandId.RunAllAction,
+        label: 'Run All Tests',
+        category: 'Test',
+        iconClass: codicon('run-all')
+    });
+
+    export const DEBUG_ALL_TESTS: Command = Command.toDefaultLocalizedCommand({
+        id: TestCommandId.DebugAllAction,
+        label: 'Debug Tests',
+        category: 'Test',
+        iconClass: codicon('debug-all')
+    });
+
+    export const RUN_TEST: Command = Command.toDefaultLocalizedCommand({
+        id: TestCommandId.RunAction,
+        label: 'Run Test',
+        category: 'Test',
+        iconClass: codicon('run')
+    });
+
+    export const RUN_TEST_WITH_PROFILE: Command = Command.toDefaultLocalizedCommand({
+        id: TestCommandId.RunUsingProfileAction,
+        category: 'Test',
+        label: 'Execute using Profile... '
```

Comment:
Minor: Too much whitespace, translation fails due to that.
```suggestion
        label: 'Execute using Profile...'
```